components/playground/main.go

```diff
@@ -374,6 +389,19 @@ func populateDefaultOpt(flagSet *pflag.FlagSet) error {
 		defaultStr(&options.TiFlashCompute.BinPath, "tiflash.compute.binpath", options.TiFlash.BinPath)
 		defaultStr(&options.TiFlashCompute.ConfigPath, "tiflash.compute.config", options.TiFlash.ConfigPath)
 		options.TiFlashCompute.UpTimeout = options.TiFlash.UpTimeout
+	case "tidb-ms":
```

Comment:
It may conflict with tikv-slim mode, it is suggested to use another flag like --pd-mode instead